--- a/boot/dtb/tegra186-quill-p3310-1000-c03-00-base.dts	2017-12-14 02:28:51.109803564 +0000
+++ b/boot/dtb/tegra186-quill-p3310-1000-c03-00-base.dts	2017-12-14 02:55:06.750988710 +0000
@@ -4038,6 +4038,16 @@
 		status = "okay";
 		linux,phandle = <0x7f>;
 		phandle = <0x7f>;
+		spidev@0 {
+			compatible = "spidev";
+			reg = <0>;
+			spi-max-frequency=<25000000>;
+		};
+		spidev@1 {
+			compatible = "spidev";
+			reg = <1>;
+			spi-max-frequency=<25000000>;
+		};
 	};
 
 	pwm@3280000 {
